src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working");
```

Comment:
The message 'Truck started working' could be more informative. It should indicate what type of Truck is working. Consider including details like the model or the specific task it's performing.